tests/test_augmentations.py

```diff
@@ -0,0 +1,118 @@
+from unittest.mock import patch
+
+import pytest
+import torch
+
+from ml4gw.augmentations import SignalInverter, SignalReverser
+
+
+@pytest.fixture(params=[0, 0.25, 1])
+def prob(request):
```

Comment:
Does this get used anywhere?

Reply:
Yes in validate_augmentors I think

Reply:
Does it? It looks like that's just getting passed the `flip_prob` argument from each of the test functions

Reply:
Done